src/js/media-chrome-button.ts

```diff
@@ -171,9 +171,9 @@ class MediaChromeButton extends globalThis.HTMLElement {
       this.nativeEl.appendChild(slotTemplate.content.cloneNode(true));
 
       this.shadowRoot.appendChild(buttonHTML);
-
-      this.tooltipEl = this.shadowRoot.querySelector('media-tooltip');
     }
+
+    this.tooltipEl = this.shadowRoot.querySelector('media-tooltip');
```

Comment:
we still want to get the media-tooltip if the shadow dom was SSR'd

Reply:
question(non-blocking): given the error, should we just move `tooltipEl` to a getter that lazy evals `this.shadowRoot.querySelector('media-tooltip');`?